src/utils/media-utils.ts

```diff
@@ -0,0 +1,51 @@
+import axios from "axios"
+
+import { config } from "@config/config"
+
+import logger from "@root/logger/logger"
+import { MediaFileInput, MediaFileOutput } from "@root/types"
+import { getAccessToken } from "@root/utils/token-retrieval-utils"
+
+const GITHUB_ORG_NAME = config.get("github.orgName")
+
+export const isMediaFileOutput = (t: MediaFileOutput): t is MediaFileOutput =>
+  (t as MediaFileOutput).sha !== undefined
+
+export const getMediaFileInfo = async ({
```

Comment:
nit: could we change the `isPrivate` to be either an optional parameter or make it optional on the type?

Reply:
Made it optional in https://github.com/isomerpages/isomercms-backend/pull/740/commits/65a108644ca42d6daa3676ed3c597be2ba13e0d3